pack.sh

```diff
@@ -63,6 +63,15 @@ for dir in $(find packages -name dist | grep -v node_modules | grep -v run-wrapp
   rsync -a $dir/ ${distdir}/
 done
 
+# Record the dependency order of NPM packages into a file
+# (This file will be opportunistically used during publishing, it does not need to be complete)
```

Comment:
Do the ones not in this file go before or after or random? Nice to include that in the comment.

Reply:
Unspecified, do not rely on any behavior there.

The details of how everything works will be documented in the PR that introduces the feature in aws-delivlib.